OneSignalSDK/onesignal/in-app-messages/src/main/java/com/onesignal/inAppMessages/internal/triggers/impl/TriggerController.kt

```diff
@@ -17,6 +17,7 @@ internal class TriggerController(
     private var _dynamicTriggerController: DynamicTriggerController,
 ) : ITriggerController, IModelStoreChangeHandler<TriggerModel> {
     val triggers: ConcurrentHashMap<String?, Any?> = ConcurrentHashMap()
+    val evaluatedCombinations = mutableSetOf<String>()
```

Comment:
This should be at the Message level, not global for all IAMs. Otherwise if new IAMs are fetched when the app is running then they won't evaluate to true when they should.

Reply:
I think this is probably fine actually, since trigger.id is a UUID.